### PR TITLE
Build error management

### PR DESCRIPTION
**Pull request by alexnoox**
- Update gulp build 
- Build break on error (not during serve)
- Refresh the theme previewer

@ouranos Please review

### CODE REVIEW

File: gulp_tasks/browsersync.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+const browserSyncDistConf = require('../conf/browsersync-dist.conf');
+
+const proxyMiddleware = require('http-proxy-middleware');
+
+browserSync.use(spa());
+
+gulp.task('browsersync', browserSyncServe);
+gulp.task('browsersync:dist', browserSyncDist);
+
+// Rewrite /admin/xxx => /xxx after the proxy
```
Comment (ouranos) [inline gulp_tasks/browsersync.js]: Copy paste? ;p

File: src/app/components/mnoe-api/organizations.svc.coffee [hunk 2]
```diff
@@ -215,17 +215,17 @@ angular.module 'mnoEnterpriseAngular'
     @can = {}
 
     _self.can.read = {
-      appInstance: (obj = null) -> _self.role.atLeastMember()
-      billing: (obj = null) -> _self.role.atLeastSuperAdmin()
-      member: (obj = null) -> _self.role.atLeastMember()
-      organizationSettings: (obj = null) -> _self.role.atLeastSuperAdmin()
+      appInstance: -> _self.role.atLeastMember()
```
Comment (ouranos) [inline src/app/components/mnoe-api/organizations.svc.coffee]: What is this? Should this be backported in the 1.1 branch?

File: src/app/stylesheets/theme.less [hunk 3]
```diff
@@ -12,40 +12,40 @@
 /*                       Twitter Bootstrap colors                        */
 /*-----------------------------------------------------------------------*/
 @brand-success:                  #33d375;
-@brand-warning:                  @mno-demo-orange;
-@brand-info:                     @mno-demo-aqua-green;
-@brand-danger:                   @mno-demo-strawberry;
-@brand-primary:                  @mno-demo-aqua-green;
+@brand-warning:                  #F2B469;
```
Comment (ouranos) [inline src/app/stylesheets/theme.less]: I thought the point was to reuse the colors defined above?

**Comment by ouranos** [comment]
I guess the correct base is `2.0`?

File: src/app/stylesheets/theme.less (hunk 3 above)
Comment (alexnoox) [inline src/app/stylesheets/theme.less]: Yes but it is not compatible with the color pickers.

Using variables:
screen shot 2017-06-21 at 4 58 18 pm

Using color hexa:
screen shot 2017-06-22 at 5 44 36 pm

File: src/app/components/mnoe-api/organizations.svc.coffee (hunk 2 above)
Comment (alexnoox) [inline src/app/components/mnoe-api/organizations.svc.coffee]: No, we don't need to backport this,I simply removed unused args.

**Pull request merged**